bin/bootstrax

```diff
@@ -1095,14 +1122,27 @@ def process_run(rd, send_heartbeats=args.production):
         if not osp.exists(dd['location']):
             fail(f"No access to live data folder {dd['location']}")
 
-        thread_info = rd.get('daq_config', dict()).get('processing_threads', dict())
-        n_readout_threads = sum([v for v in thread_info.values()])
         try:
-            daq_chunk_duration = int(rd['daq_config']['strax_chunk_length'] * 1e9)
-            daq_overlap_chunk_duration = int(rd['daq_config']['strax_chunk_overlap'] * 1e9)
+            # Fetch parameters from the rundoc. If not readably, let's use redax' default
+            # values (that are hardcoded here).
+            if 'daq_conig' not in rd:
+                log_warning('No daq_config in the rundoc!')
```

Comment:
I'd argue that this should be a fatal error rather than a warning.

Reply:
will change it